test/kuttl/tests/notifications/03-assert.yaml

```diff
@@ -0,0 +1,7 @@
+# Verify notifications transporturl was cleaned up (command should fail)
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      oc get transporturl/notifications-neutron-transport -n $NAMESPACE || exit 0
```

Comment:
iiuc you want to check that the notirifations-neutron-transport cr is gone. you could do this using an errors file, like https://github.com/openstack-k8s-operators/openstack-operator/blob/main/tests/kuttl/tests/ctlplane-tls-custom-route/04-errors-cleanup.yaml, to check that a resource is gone and you expect an error.